qa/smoke-test-http/src/javaRestTest/java/org/elasticsearch/http/BlockedSearcherRestCancellationTestCase.java

```diff
@@ -134,6 +138,12 @@ public List<Setting<?>> getSettings() {
 
     private static class SearcherBlockingEngine extends ReadOnlyEngine {
 
+        // using a specialized logger for this case because and "logger" means "Engine#logger"
```

Comment:
"because" as in "just because", or is there something missing from the sentence? :)))

Reply:
I'd assume an implicit "just" everywhere 😉